toolchain/binutils.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Hello @jayarama-ibm, I haven't found this in Autotest, so did you implement this test, or is it port from other test-framework? I'm asking because there needs to be the license header with author and/or copyright. So please add GPL compatible header and the author (either you or link to the original source).

Please see https://github.com/avocado-framework/avocado/blob/master/avocado/utils/software_manager.py for details.